web/client/reducers/mapInfo.js

```diff
@@ -27,7 +27,7 @@ const {RESET_CONTROLS} = require('../actions/controls');
 
 const assign = require('object-assign');
 const {head} = require('lodash');
-const {inside} = require('turf');
+const {intersect, buffer} = require('turf');
```

Comment:
We should use:

turf/node_modules/turf-intersect

and 

turf/node_modules/turf-buffer

to avoid including the complete turf bundle